xep-0367.xml

```diff
@@ -109,42 +117,67 @@
-    Clients and servers MUST NOT include an &lt;attach-to/&gt; element on
-    messages with a non-messaging payload unless they are including it on an
-    error which may be attached to the message that caused the error to be
-    generated.
-  </p>
+  <section2 topic='Using the correct ID' anchor='business-id'>
+    <p>
+      For messages of type 'groupchat', the stanza's 'id' attribute MUST NOT be
+      used for attachments. Instead, in group chat situations, the ID assigned to
+      the stanza by the group chat itself must be used. This is discovered in a
+      &lt;stanza-id&gt; element with a 'by' attribute that matches the bare JID of
+      the group chat, as defined in &xep0359;.
+    </p>
+    <p>
+      For other message types the attacher should use the 'id' from a XEP-0359
+      &lt;origin-id&gt; if present, or the value of the 'id' attribute on the
+      &lt;message&gt; otherwise.
+    </p>
+  </section2>
+  <section2 topic='Display of attached messages' anchor='business-display'>
+    <p>
+      A receiving client MAY choose to show the attached message next to or below
+      the indicated message in whatever display is used for messages or can
+      choose to display the attachment in another way (including as a normal
+      message, completely ignoring the attach-to element).
+    </p>
+    <p>
+      A receiving client SHOULD indicate that the message is an attachment, and
+      not a part of the original message to prevent confusion.
+    </p>
+  </section2>
+  <section2 topic='General' anchor='business-general'>
+    <p>
+      &lt;attach-to/&gt; elements MUST NOT be put on any stanza type other than
+      messages.
+    </p>
+    <p>
+      A server may choose to strip some &lt;attach-to/&gt; messages based on
+      local policy (eg. a server might have a policy that only it can create
+      message attachments).
+    </p>
+    <p>Clients MUST send ids on messages if they support attachments.</p>
+    <p>
+      Messages MUST NOT contain more than one &lt;attach-to/&gt; element.
+    </p>
+    <p>
+      Clients and servers MUST NOT include an &lt;attach-to/&gt; element on
+      messages with a non-messaging payload unless they are including it on an
+      error which may be attached to the message that caused the error to be
+      generated.
+    </p>
+   </section2>
 </section1>
 <section1 topic='Security Considerations' anchor='security'>
   <p>
-    Clients that implement message attachments MUST be careful not to display
-    the attachments in such a way that they could be confused with the original
-    message and cause someone viewing the conversation to assume they were sent
-    by the sender of the message being attached to.
+    Clients that implement message attachments MUST display the attachments
+    in such a way that they could be confused with the original message and
+    cause someone viewing the conversation to assume they were sent by the
+    sender of the message being attached to.
```

Comment:
@mwild1 Is there a "not" missing here?

I.e. "in such a way that they could NOT be confused with the original message"

Reply:
@jcbrand @SamWhited Ha, sorry for the late response. Yes, obviously there is a 'not' missing :)